build.mjs

```diff
@@ -36,7 +36,7 @@ const ES_BUILD_OPTIONS = {
     "uuid",
     "hex",
     "os",
-    "fs/promises",
```

Comment:
Don't we need to specify both `fs/promises` and `fs`? as they are separate modules

Reply:
`fs/promises` will be excluded by default as the child path of `fs`.
Trust me, it was my contribution to `esbuild`.
https://github.com/evanw/esbuild/pull/186